src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int sideA;
+    private int sideB;
+    private int height;
+
+    public IsoscelesTrapezoid(int sideA, int sideB, int height) {

```

Comment:
```suggestion
    public IsoscelesTrapezoid(String color, int sideA, int sideB, int height) {
        super(color);
```
change in each figure